src/utils/EventBus.ts

```diff
@@ -0,0 +1,35 @@
+export class EventBus {
+  listeners: {
+    [event: string]: any[];
```

Comment:
Нужно более точно типизировать подписчиков. Подписчиком является функция, принимающая n-ое количество аргументов. Можно написать так
```suggestion
    [event: string]: Array<(...unknown[]) => void>;
```